### PR TITLE
Clarify the instructions for the "Unit Testing" section of the contributing guidelines.

### PR DESCRIPTION
**Pull request by stevegrunwell**
After a bit of a run-around this morning trying to get unit testing working (see #766), I wanted to help clean up the documentation and make it very clear what needs to be done to get unit tests working in a local copy of the repository.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,13 +12,19 @@ Once a commit is made to `develop`, a PR should be opened from `develop` into `m
 
 # Unit Testing
 
-TL;DR
+When contributing to WPCS, it's recommended that the project has its own copy of `phpcs`, installed via Composer:
```
Comment (GaryJones) [inline CONTRIBUTING.md]: I've never actually thought about just using the dependency version of PHPCS for running tests / submitting PRs upstream to PHPCS itself.

Comment (jrfnl) [inline CONTRIBUTING.md]: This is only valid if your preferred method of installation is Composer. The PEAR install comes with the test suite already and if you also develop for PHPCS itself, you probably have a stand-alone PHPCS git clone somewhere which comes with the test suite as well.

**Comment by GaryJones** [comment]
@stevegrunwell 
After the change to support PHP 3.*, the instructions for how to run unit tests were changed, and clarifications added. You can see the current content at https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/.github/CONTRIBUTING.md#unit-testing.

Do you feel any more clarifications need to be made (in regards to the intent of this PR)?

**Comment by stevegrunwell** [comment]
I think the latest version of the contributing doc captures the essence of what I was trying to get at with this PR, thank you!

**Pull request closed without merging**